src/main/java/com/example/hello/ServletInitializer.java

```diff
@@ -3,6 +3,7 @@
 import org.springframework.boot.builder.SpringApplicationBuilder;
 import org.springframework.boot.web.servlet.support.SpringBootServletInitializer;
 
+//add
```

Comment:
why do you need this comment here?
if it is not necessary please remove
and make changes to all "add" comments